src/test_all.py

```diff
@@ -33,6 +35,18 @@
 def assert_language_name(name):
     assert name in language_names, f"Did not find `{name}` among iso639 language names: {language_names}"
 
+def skewness(result):
+    value,counts = np.unique(result, return_counts=True)
+    norm_counts = counts / counts.sum()
+    entropy=-(norm_counts * np.log(norm_counts)/np.log(len(value))).sum()
+    assert entropy > 0.7, "Classes distribution is skewed"
+
+def skewness2(result):
+    value,counts = np.unique(result, return_counts=True)
+    average=np.average(counts)
+    metric=np.average(abs(counts-average)/counts.sum())
+    assert metric < 0.2, "Classes distribution is skewed"
```

Comment:
For now, suggest just printing the skew, rather than failing. We'll need to pick a threshold based on the outcome.

Reply:
I wasn't sure which metric would be better. Should I print the skew based on the both metrics?

Reply:
Yes! print them side by side. Also print the distribution as well.